action.yml

```diff
@@ -25,6 +25,14 @@ inputs:
     description: 'The branch, tag, or SHA of the repository to clone'
     required: false
 
+  checkout-repository:
+    description: 'The repository to checkout if not the repository that triggered the action. For use when building GitHub Apps'
+    required: false
+
+  checkout-token:
+    description: 'Token to use for checkout if checking out a repository out of scope for GITHUB_TOKEN'
+    required: false
```

Comment:
I will try with default value as: `${{ github.token }}`

Reply:
will update tomorrow after testing.

Reply:
updated with this default